src/styles/variables.js

```diff
@@ -47,5 +49,14 @@ export default {
     tooltipzIndex: 10050,
     gutterWidth: 16,
     popoverMenuShadow: '0px 4px 12px 0px rgba(0, 0, 0, 0.06)',
+    lineHeightSmall: PixelRatio.getFontScale() * 14 > 16 ? 16 : 14 * PixelRatio.getFontScale(),
+    lineHeightNormal: PixelRatio.getFontScale() * 16 > 21 ? 21 : 16 * PixelRatio.getFontScale(),
+    lineHeightLarge: PixelRatio.getFontScale() * 18 > 24 ? 24 : 18 * PixelRatio.getFontScale(),
+    lineHeightXLarge: PixelRatio.getFontScale() * 20 > 22 ? 22 : 20 * PixelRatio.getFontScale(),
```

Comment:
This doesn't look dry & the code is not understandable. Can we have a method with proper comments to make compute these values?

Reply:
Done, I know we are putting our methods inside `StyleUtils.js`, but if we import `StyleUtils` inside `Variables.js` we will get this lint error `Dependency cycle detected`, so I put it inside `Variables.js`

Reply:
waiting for @marcaaron decision.